src/common/config.h

```diff
@@ -25,6 +25,7 @@
 #define OSD_REP_SPLAY   1
 #define OSD_REP_CHAIN   2
 
+#define VAL1 1
```

Comment:
@tchaikov Thanks for the suggestion. I wanted someone else to review this cleanup.